.github/ISSUE_TEMPLATE/2-account-issue.yml

```diff
@@ -0,0 +1,50 @@
+name: '👤 Account issue'
+labels: ['triage']
+description: An issue with your Jenkins account
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before hitting the button, please answer these questions.
+        It's helpful to search the existing GitHub issues first.
+        There is a chance that another user has already reported the issue you're facing, or it's a known issue that we're already aware of. 
+
+        Fill in as much of the template below as you can.
+        The more information we have the better we can help you.
+
+        You can reset the password of your Jenkins account (issues.jenkins.io, *.jenkins.io websites) by visiting the page accounts.jenkins.io and clicking on "reset my password".
```

Comment:
What do you mean?

Reply:
![Screenshot 2022-11-15 at 16 21 06](https://user-images.githubusercontent.com/13383509/201957047-929cfa7e-ab76-46cf-9632-7838eb43edaa.png)

There's a "Forgot password?" button, rather than "reset my password"

Reply:
OK.
Not sure if we should link to https://accounts.jenkins.io/ with the "forgot password", or if we should directly link to the https://accounts.jenkins.io/passwordReset page.
WDYT?

Reply:
https://accounts.jenkins.io/login is fine imo.

Reply:
https://account.jenkins.io is fine to me